src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -48,7 +49,7 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('console.xml');
         }
 
-        if (!interface_exists('Symfony\Component\Translation\TranslatorInterface')) {
+        if (!interface_exists(TranslatorInterface::class)) {
```

Comment:
this change is wrong. We still need to detect the translation component itself to keep the extension, not the contracts component (you could have it without having the translation component)

Reply:
good catch, fixed thanks